src/FlexVotingClient.sol

```diff
@@ -92,7 +92,7 @@ abstract contract FlexVotingClient {
   /// token that `_user` has claim to in this system. It may or may not be
   /// equivalent to the withdrawable balance of `GOVERNOR`s token for `user`,
   /// e.g. if the internal representation of balance has been scaled down.
-  function _rawBalanceOf(address _user) internal view virtual returns (uint256);
```

Comment:
Changed because there is no `Trace256`. Should we write one for OZ?

Reply:
Traces are a list of checkpoint. Each checkpoint contains:
- a key
- a value
such that sizeof(key) + sizeof(value) = 256

- `Trace224` is `uint224` for value, and `uint32` for key
- `Trace208` is `uint208` for value, and `uint48` for key

`Trace256` would not have any room for the key ...

Reply:
Switched to Trace208, thanks!